src/Pyz/Zed/DataImport/Business/Model/ProductAbstract/ProductAbstractHydratorStep.php

```diff
@@ -333,4 +333,24 @@ protected function getCategoryProductOrder($categoryProductOrder): array
 
         return array_map('trim', $categoryProductOrder);
     }
+
+    /**
+     * @param array<mixed> $attributes
+     *
+     * @return array<mixed>
+     */
+    protected function formatMultiSelectProductAttributes(array $attributes): array
+    {
+        foreach ($attributes as $key => $value) {
+            if (is_string($value) && preg_match('/^\[.*\]$/', $value)) {
+                $json = str_replace("'", '"', $value);
+                $decoded = json_decode($json, true);
```

Comment:
not sure if it is important for project level but we usually use UtilEncodingService (relevant to all the integration PRs)

Reply:
You're right, for the project level it's not relevant (you could check other files).